src/main/aerospike.c

```diff
@@ -138,7 +138,7 @@ static int Aerospike_Clear(PyObject *aerospike)
 PyMODINIT_FUNC PyInit_aerospike(void)
 {
 
-    const char version[] = "14.1.0";
+    const char version[] = "14.2.0-rc.1";
```

Comment:
I'm assuming this is going to be changed to 14.2.0 by the pipeline later?

Reply:
Yes I will bump the version to a release version after I merge this PR